activerecord/test/cases/primary_keys_test.rb

```diff
@@ -351,9 +351,9 @@ class Widget < ActiveRecord::Base
     test "schema dump primary key with bigserial" do
       schema = dump_table_schema "widgets"
       if current_adapter?(:PostgreSQLAdapter)
-        assert_match %r{create_table "widgets", id: :bigserial}, schema
+        assert_match %r{create_table "widgets", id: :bigserial, force: :cascade}, schema
```

Comment:
Is this now attempting to match the whole line instead of the partial? or is `force: :cascade` being added after this change?